controllers/user/login.js

```diff
@@ -19,14 +19,15 @@ const login = async (req, res, next) => {
     let { username } = req.body;
     const { password } = req.body;
     username = username.toLowerCase();
-    const userExist = await models.User.findOne({ username });
+    const userExist = await models.User.findOne({ username }).lean();
 
     if (userExist) {
       const compare = bcrypt.compareSync(password, userExist.password);
 
       const user = await models.User.findOne({ username }).select([
         '-password'
-      ]);
+      ]).lean();
```

Comment:
With `lean()` used on line 22. We can rename that variable to `user` and instead of doing another DB call on line 27, we can just `select(['-password'])` from `user` on line 22 directly. 

We will avoid one DB call and deleting the PW

Reply:
Solved